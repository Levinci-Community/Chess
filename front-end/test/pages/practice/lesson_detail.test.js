import React from "react";
import { render } from "@testing-library/react";
import LessonDetailPage from "../../../src/pages/practice/lesson_detail";
import { BrowserRouter as Router } from "react-router-dom";

jest.mock("react-i18next", () => ({
    useTranslation: () => ({
      t: (key) => key,
    }),
  }));
  
  test("renders LessonDetailPage snapshot", () => {
    const { container } = render(
      <Router>
        <LessonDetailPage />
      </Router>
    );
    expect(container).toMatchSnapshot();
  });