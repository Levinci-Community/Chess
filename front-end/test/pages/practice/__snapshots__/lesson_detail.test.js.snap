// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders LessonDetailPage snapshot 1`] = `
<div>
  <div
    class="chakra-container css-3laa41"
  >
    <h1
      class="chakra-heading css-1itv5e3"
    >
      Loading...
    </h1>
  </div>
</div>
`;
